pages/3_manage_giftcard.php

```diff
@@ -0,0 +1,77 @@
+<?php
+//Set the title of the page
+setTitle("Beheer cadeaubonnen");
+
+//Retrieve all current giftcards.
+$giftcards = base_query("SELECT * FROM coupon ORDER BY CurrentValue DESC");
```

Comment:
Het is beter om dit na lijn 35 te doen zodat eventuele veranderingen ook direct te zien zijn.